src/state/bidib_state_setter.c

```diff
@@ -449,9 +490,10 @@ void bidib_state_lc_stat(t_bidib_node_address node_address, t_bidib_peripheral_p
 			                "Aspect 0x%02x of peripheral %s is not mapped in config files",
 			                portstat, peripheral_mapping->id->str);
 		} else {
-			syslog_libbidib(LOG_INFO,
-			                "Feedback for action id %d: Peripheral: %s has aspect: %s (0x%02x)",
-			                action_id, peripheral_mapping->id->str, aspect_mapping->id->str, portstat);
+			// Temporarily disabled for debugging (less logs to read)
```

Comment:
Please resolve

Reply:
Changed back to normal. However, I set the logging level to DEBUG, as these messages we see there are not so important and very frequent (peripheral "sync" has aspect... gets received very often -> lots of logs on the info level.)